docs/12.0/guides/cell-features/disabled-cells.md

```diff
@@ -18,7 +18,7 @@ Disabling a cell makes the cell read-only or non-editable. Both have similar out
 
 ## Read-only columns
 
-In many use cases, you will need to configure a certain column to be read-only. This column will be available for keyboard navigation and <kbd>CTRL+C</kbd>. Editing and pasting data will be disabled.
+In many use cases, you will need to configure a certain column to be read-only. This column will be available for keyboard navigation and <kbd>**Ctrl**</kbd> + <kbd>**C**</kbd>. Editing and pasting data will be disabled.
```

Comment:
I think you forgot to add `coping`.

Reply:
@aninde I didn't edit the descriptions, but sure, it should be there: [5de8328](https://github.com/handsontable/handsontable/pull/9448/commits/5de83281ab61c8bcdeb0b1bf783219bfe04c2402)